service/src/main/java/bio/terra/pipelines/stairway/imputation/SubmitCromwellRunSetStep.java

```diff
@@ -66,112 +72,17 @@ public StepResult doStep(FlightContext flightContext)
               "couldn't find method version id for method name " + wdlMethodName));
     }
 
-    // input definitions - hardcoded for now
-    // in future this will be pulled from the workspace
-    String workspaceStorageContainerUri =
-        "https://lz8b0d07a4d28c13150a1a12.blob.core.windows.net/sc-94fd136b-4231-4e80-ab0c-76d8a2811066";
-
-    // in future these will be pulled from the working map
-    List<String> contigsInputValue =
-        new ArrayList<>(
-            Arrays.asList(
-                "chr1", "chr2", "chr3", "chr4", "chr5", "chr6", "chr7", "chr8", "chr9", "chr10",
-                "chr11", "chr12", "chr13", "chr14", "chr15", "chr16", "chr17", "chr18", "chr19",
-                "chr20", "chr21", "chr22"));
-    String refDictInputValue = workspaceStorageContainerUri + "/hg38/Homo_sapiens_assembly38.dict";
-    String geneticMapsPathInputValue =
-        workspaceStorageContainerUri + "/plink-genetic-maps/GRCh38_fixed/";
-    String referencePanelPathInputValue =
-        workspaceStorageContainerUri + "/hg38/Homo_sapiens_assembly38.dict";
-
     // prepare cbas submission (run set) object
-
     // this is mostly a manually generated run set request definition, we'll want to be able to auto
```

Comment:
can remove this comment and the one under it!  Or update it to saw inputs are generated and outputs arent yet generated

Reply:
whoops good catch, thanks!